test_conformance/generic_address_space/main.cpp

```diff
@@ -106,6 +107,21 @@ test_status InitCL(cl_device_id device) {
         {
             return TEST_SKIP;
         }
+
+        error = clGetDeviceInfo(device, CL_DEVICE_MAX_GLOBAL_VARIABLE_SIZE,
+                                sizeof(max_gvar_size), &max_gvar_size, NULL);
+        if (error != CL_SUCCESS)
+        {
+            print_error(error,
+                        "Unable to MAX_GLOBAL_VARIABLE_SIZE value. "
+                        "Program scope variable support is required");
```

Comment:
Nitpick: the current phrasing for this error message is confusing.  How about:

```suggestion
            print_error(error,
                        "Unable to query CL_DEVICE_MAX_GLOBAL_VARIABLE_SIZE.");
```